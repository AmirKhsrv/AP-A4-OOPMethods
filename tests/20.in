recommend_second_approach 22
credit 86
best_reviewer
credit 55
credit 13
best_reviewer
credit 105
show_sorted_shelf 65 currently_reading Fiction
credit 75
show_author_info 1225
recommend_first_approach 72
best_reviewer
recommend_first_approach 7
best_book
best_reviewer
recommend_first_approach 28
