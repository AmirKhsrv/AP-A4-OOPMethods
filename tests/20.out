id: 303
Title: The Good Luck Charm
Genre: Romance
Author: Helena Hunting
Reviews:
id: 99 Rating: 3 Likes: 8 Date: November 16 2018
id: 142 Rating: 4 Likes: 0 Date: January 20 2019
id: 949 Rating: 1 Likes: 1 Date: May 14 2017
id: 1209 Rating: 1 Likes: 0 Date: August 22 2013
id: 1350 Rating: 5 Likes: 2 Date: September 25 2018
id: 1595 Rating: 1 Likes: 0 Date: July 1 2013
id: 1747 Rating: 1 Likes: 0 Date: November 13 2018
id: 1846 Rating: 4 Likes: 0 Date: July 9 2016
id: 3401 Rating: 3 Likes: 0 Date: January 1 2016
id: 3889 Rating: 5 Likes: 0 Date: October 17 2017
id: 4887 Rating: 5 Likes: 3 Date: May 4 2018
id: 5127 Rating: 5 Likes: 0 Date: November 9 2010
id: 5574 Rating: 3 Likes: 0 Date: October 6 2014
id: 6137 Rating: 3 Likes: 0 Date: June 28 2017
id: 6480 Rating: 1 Likes: 1 Date: May 1 2011
id: 7559 Rating: 0 Likes: 7 Date: August 19 2017
id: 7913 Rating: 3 Likes: 0 Date: July 10 2019
id: 8048 Rating: 2 Likes: 0 Date: December 23 2018
id: 8160 Rating: 1 Likes: 0 Date: February 26 2017
id: 8310 Rating: 2 Likes: 0 Date: November 29 2019
id: 8626 Rating: 5 Likes: 0 Date: June 16 2014
id: 8683 Rating: 4 Likes: 0 Date: June 20 2013
id: 8858 Rating: 0 Likes: 0 Date: December 16 2015
id: 8862 Rating: 4 Likes: 0 Date: August 26 2018
id: 8903 Rating: 5 Likes: 0 Date: July 11 2018
id: 9182 Rating: 2 Likes: 0 Date: October 12 2018
id: 10427 Rating: 2 Likes: 0 Date: July 28 2018
id: 10432 Rating: 3 Likes: 0 Date: January 9 2018
id: 10490 Rating: 4 Likes: 0 Date: October 20 2017
id: 11047 Rating: 2 Likes: 24 Date: October 15 2010
id: 12102 Rating: 2 Likes: 3 Date: April 22 2018
id: 12147 Rating: 4 Likes: 2 Date: December 12 2019
id: 12633 Rating: 3 Likes: 0 Date: March 21 2016
id: 13302 Rating: 4 Likes: 0 Date: April 18 2015
id: 14111 Rating: 3 Likes: 1 Date: September 14 2016
id: 14698 Rating: 2 Likes: 0 Date: October 22 2019
id: 16634 Rating: 2 Likes: 0 Date: October 13 2018
0.007221
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
0.005665
0.007470
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
0.004044
id: 40
Title: The Master and Margarita
Genre: Fiction
Author: Mikhail Bulgakov
***
id: 60
Title: Annie on My Mind
Genre: Lgbt
Author: Nancy Garden
***
id: 208
Title: Brigitta of the White Forest
Genre: Fantasy
Author: Danika Dinsmore
***
id: 154
Title: Empty Heart: False Love
Genre: Sequential Art
Author: Masara Minase
***
id: 91
Title: Farsighted
Genre: Fantasy
Author: Emlyn Chand
***
id: 41
Title: If You Find Me
Genre: Young Adult
Author: Emily Murdoch
***
id: 266
Title: Madhur Jaffrey's World Vegetarian: More Than 650 Meatless Recipes from Around the World
Genre: Food and Drink
Author: Madhur Jaffrey
***
id: 237
Title: North and South
Genre: Classics
Author: Elizabeth Gaskell
***
id: 396
Title: Tao Te Ching
Genre: Philosophy
Author: Lao Tzu
***
id: 160
Title: The Lies of Locke Lamora
Genre: Fantasy
Author: Scott Lynch
***
id: 258
Title: Tiger's Voyage
Genre: Fantasy
Author: Colleen Houck
***
0.007022
id: 1225
Name: Kate Wilhelm
Year of Birth: 1985
Place of Birth: Fiji
Member Since: October 5 2016
Genres: Science Fiction, Thriller
Books:
id: 82
Title: Genghis Khan and the Making of the Modern World
Genre: History
Author: Jack Weatherford
Reviews:
id: 245 Rating: 1 Likes: 1103 Date: February 12 2011
id: 1978 Rating: 3 Likes: 0 Date: May 26 2009
id: 2419 Rating: 4 Likes: 0 Date: June 4 2018
id: 2503 Rating: 3 Likes: 0 Date: January 7 2019
id: 2563 Rating: 5 Likes: 0 Date: June 27 2016
id: 3171 Rating: 0 Likes: 0 Date: August 13 2015
id: 3338 Rating: 5 Likes: 0 Date: December 21 2017
id: 3556 Rating: 5 Likes: 0 Date: March 29 2018
id: 3774 Rating: 0 Likes: 0 Date: October 1 2011
id: 3883 Rating: 1 Likes: 0 Date: July 24 2019
id: 4155 Rating: 3 Likes: 3 Date: March 17 2017
id: 4772 Rating: 4 Likes: 0 Date: January 11 2018
id: 5291 Rating: 1 Likes: 0 Date: October 18 2019
id: 5481 Rating: 0 Likes: 0 Date: June 27 2019
id: 5674 Rating: 1 Likes: 0 Date: October 13 2018
id: 6473 Rating: 5 Likes: 4 Date: December 3 2014
id: 6568 Rating: 4 Likes: 3 Date: December 6 2018
id: 6572 Rating: 5 Likes: 6 Date: May 7 2015
id: 6611 Rating: 1 Likes: 2 Date: October 18 2018
id: 7436 Rating: 0 Likes: 2 Date: May 10 2017
id: 7886 Rating: 5 Likes: 0 Date: June 3 2018
id: 9102 Rating: 5 Likes: 0 Date: January 15 2014
id: 9385 Rating: 5 Likes: 0 Date: December 5 2019
id: 9645 Rating: 3 Likes: 0 Date: May 22 2019
id: 10158 Rating: 3 Likes: 0 Date: August 26 2014
id: 10628 Rating: 4 Likes: 1 Date: March 3 2019
id: 10638 Rating: 0 Likes: 0 Date: July 21 2012
id: 11073 Rating: 1 Likes: 0 Date: February 8 2018
id: 12096 Rating: 0 Likes: 1 Date: April 12 2018
id: 12761 Rating: 4 Likes: 0 Date: June 14 2019
id: 13298 Rating: 2 Likes: 0 Date: January 2 2018
id: 13434 Rating: 2 Likes: 0 Date: March 8 2017
id: 13684 Rating: 5 Likes: 0 Date: December 27 2019
id: 13932 Rating: 1 Likes: 0 Date: July 28 2014
id: 14519 Rating: 1 Likes: 0 Date: March 18 2016
id: 14653 Rating: 2 Likes: 1 Date: March 28 2013
id: 15057 Rating: 5 Likes: 0 Date: May 7 2017
id: 15171 Rating: 4 Likes: 0 Date: October 17 2011
id: 15763 Rating: 0 Likes: 1 Date: October 27 2018
id: 15815 Rating: 0 Likes: 0 Date: September 3 2017
id: 15968 Rating: 4 Likes: 2 Date: June 12 2018
id: 16595 Rating: 2 Likes: 0 Date: November 15 2018
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 392
Title: From the Mixed-Up Files of Mrs. Basil E. Frankweiler
Genre: Fiction
Author: E.L. Konigsburg
Reviews:
id: 356 Rating: 3 Likes: 0 Date: August 24 2018
id: 396 Rating: 2 Likes: 558 Date: January 16 2011
id: 677 Rating: 5 Likes: 3 Date: August 13 2018
id: 914 Rating: 5 Likes: 0 Date: June 3 2019
id: 1654 Rating: 0 Likes: 0 Date: February 6 2019
id: 1665 Rating: 5 Likes: 0 Date: November 21 2017
id: 1866 Rating: 0 Likes: 0 Date: December 27 2018
id: 2173 Rating: 2 Likes: 0 Date: February 17 2012
id: 2285 Rating: 5 Likes: 0 Date: March 28 2013
id: 2901 Rating: 5 Likes: 0 Date: July 20 2017
id: 4481 Rating: 5 Likes: 2 Date: May 21 2016
id: 5827 Rating: 4 Likes: 0 Date: November 25 2019
id: 5975 Rating: 1 Likes: 0 Date: March 28 2018
id: 6899 Rating: 5 Likes: 1 Date: February 27 2018
id: 8914 Rating: 1 Likes: 0 Date: July 12 2013
id: 9248 Rating: 5 Likes: 0 Date: October 4 2018
id: 9999 Rating: 5 Likes: 0 Date: March 18 2015
id: 10317 Rating: 4 Likes: 0 Date: April 18 2014
id: 10521 Rating: 4 Likes: 0 Date: September 13 2019
id: 10822 Rating: 5 Likes: 0 Date: May 11 2016
id: 11689 Rating: 5 Likes: 1 Date: January 27 2017
id: 12274 Rating: 5 Likes: 0 Date: August 25 2015
id: 12379 Rating: 5 Likes: 0 Date: June 23 2015
id: 12431 Rating: 0 Likes: 1 Date: July 25 2019
id: 12626 Rating: 3 Likes: 0 Date: September 18 2019
id: 13444 Rating: 0 Likes: 0 Date: June 26 2013
id: 13813 Rating: 3 Likes: 0 Date: May 27 2018
id: 14083 Rating: 3 Likes: 0 Date: October 26 2014
id: 14142 Rating: 4 Likes: 0 Date: December 27 2014
id: 14199 Rating: 3 Likes: 4 Date: October 12 2015
id: 14427 Rating: 1 Likes: 2 Date: June 15 2018
id: 16431 Rating: 5 Likes: 0 Date: October 28 2014
id: 16509 Rating: 5 Likes: 0 Date: March 17 2016
id: 16740 Rating: 3 Likes: 0 Date: April 26 2016
id: 16829 Rating: 0 Likes: 0 Date: October 15 2017
id: 17340 Rating: 3 Likes: 0 Date: November 13 2017
id: 405
Title: Ellen Foster
Genre: Fiction
Author: Kaye Gibbons
Average Rating: 3.42
id: 31
Name: Athena Souri
Place of Birth: Switzerland
Member Since: December 24 2010
Favorite Genres: Romance
Favorite Authors: Carl Sagan, Colleen Clayton, Greer Noble
Number of Books in Read Shelf: 8
Number of Books in Want to Read Shelf: 15
Number of Books in Currently Reading Shelf: 6
Number of Likes: 2455
id: 1
Title: Forging Zero
Genre: Science Fiction
Author: Sara  King
Reviews:
id: 416 Rating: 2 Likes: 1 Date: April 4 2014
id: 708 Rating: 4 Likes: 0 Date: October 19 2010
id: 866 Rating: 0 Likes: 0 Date: July 5 2016
id: 889 Rating: 1 Likes: 0 Date: December 21 2017
id: 1572 Rating: 2 Likes: 0 Date: April 18 2015
id: 1766 Rating: 2 Likes: 0 Date: June 8 2013
id: 2152 Rating: 1 Likes: 3 Date: January 2 2019
id: 2948 Rating: 1 Likes: 0 Date: October 14 2017
id: 3017 Rating: 4 Likes: 0 Date: July 25 2018
id: 4267 Rating: 0 Likes: 1 Date: January 14 2015
id: 4640 Rating: 1 Likes: 0 Date: September 9 2016
id: 5212 Rating: 4 Likes: 0 Date: January 12 2010
id: 5554 Rating: 5 Likes: 0 Date: January 27 2015
id: 6186 Rating: 0 Likes: 0 Date: July 14 2018
id: 6356 Rating: 5 Likes: 3 Date: December 13 2015
id: 6574 Rating: 4 Likes: 9 Date: March 15 2018
id: 7151 Rating: 3 Likes: 5 Date: July 18 2014
id: 7884 Rating: 3 Likes: 0 Date: August 29 2013
id: 9223 Rating: 3 Likes: 0 Date: October 20 2011
id: 9810 Rating: 2 Likes: 0 Date: June 5 2016
id: 11056 Rating: 2 Likes: 0 Date: October 16 2012
id: 11581 Rating: 4 Likes: 4 Date: July 24 2015
id: 11997 Rating: 5 Likes: 3 Date: November 19 2015
id: 12488 Rating: 4 Likes: 0 Date: September 17 2019
id: 12559 Rating: 5 Likes: 0 Date: March 5 2010
id: 13030 Rating: 2 Likes: 2 Date: November 23 2017
id: 13083 Rating: 2 Likes: 0 Date: August 8 2019
id: 13266 Rating: 2 Likes: 0 Date: January 7 2015
id: 13897 Rating: 1 Likes: 0 Date: February 28 2014
id: 15083 Rating: 5 Likes: 0 Date: July 24 2018
id: 16304 Rating: 0 Likes: 0 Date: August 10 2018
